dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -133,7 +133,7 @@
     QUERY_TASK_INSTANCE_LOG_ERROR(10103, "view task instance log error", "查询任务实例日志错误"),
     DOWNLOAD_TASK_INSTANCE_LOG_FILE_ERROR(10104, "download task instance log file error", "下载任务日志文件错误"),
     CREATE_PROCESS_DEFINITION(10105, "create process definition", "创建工作流错误"),
-    VERIFY_PROCESS_DEFINITION_NAME_UNIQUE_ERROR(10106, "verify process definition name unique error", "工作流定义名称已存在"),
```

Comment:
There is essentially no difference between the `VERIFY_PROCESS_DEFINITION_NAME_UNIQUE_ERROR` and `PROCESS_DEFINITION_NAME_EXIST`, and it's not necessary to add an additional error code.
The easiest way is just renaming `VERIFY_PROCESS_DEFINITION_NAME_UNIQUE_ERROR ` to another